src/core/src/Context.cpp

```diff
@@ -384,6 +379,11 @@ void Context::processCesiumGlobeAnchorChanged(const cesium::omniverse::ChangedPr
     }
 }
 
+[[maybe_unused]] void
+Context::processCesiumIonServerChanged([[maybe_unused]] const cesium::omniverse::ChangedPrim& changedPrim) {
+    // TODO: I have no idea what to do here.
```

Comment:
Possibly:

* Watch for changes on all attributes. Reload tileset and imagery that reference this server.
* Update `SessionRegistry` if USD path changes
* Double check if any other data structures need to be updated (e.g. anything keyed based on ion url?)